### PR TITLE
Syntax and Grammar Edits

### PR DESCRIPTION
**Pull request by chrispycheng**
- [x] 00-Overview
- [x] 01-shell-basics
- [x] 02-vcs
- [x] 03-packages
- [x] 04-legalities
- [x] 05-misc
- [x] 11-wrap-up
- [x] Final review of content issues, creative adjustments, etc.

### CODE REVIEW

File: _episodes/00-Overview.md
Comment (yarikoptic) [inline _episodes/00-Overview.md]: `to first to skim`?

**Review by yarikoptic — COMMENTED** [review]
Thank you 1 lot @chrispycheng !  Left a single comment since was not sure if the fix needs a fix ;)

I guess, from the checklist, I will wait for more edits in other lessons before we actually merge it, right?

File: _episodes/00-Overview.md
Comment (chrispycheng) [reply _episodes/00-Overview.md]: Corrected, thank you! Yup I was thinking it'd be cool to drop all the edits at once in one pull request. Or is it possible to incorporate suggested edits in a pull request several times?

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: nope, graphical -- https://en.wikipedia.org/wiki/Graphical_user_interface

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: better even make it a single word `builtin` since that is how `man bash` etc use it

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: what I wanted to say was `Many FSL scripts use **sh**` (not that it sh somehow has FSL scripts).  FSL is a software package in neuroimaging

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: capitalization was to show what `ksh` abbreviation is based on

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: well -- `specifying` is kinda out of place since it is a question "how to determine" not to specify

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: keep original `"` doublequotes

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: again built-in, so unify to just be builtin everywhere please

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md:257]: so `…` because there is more text to the question? might be nice indeed - anywhere else I use such form?

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: `,,`

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: well -- joining like that confuses the meaning of the 2nd item into the first. Even without writing a script your could use history to determine which command you ran. Please retain the items or rephrase/split to make it clear

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: I think the quote in`“test` became a different/fancy one.

**Review by yarikoptic — CHANGES_REQUESTED** [review]
Yeay, thanks, good changes!!!  I left some comments at a few spots where I think it needs a bit of fixing ;-)
Cheers!

File: _episodes/01-shell-basics.md
Comment (chrispycheng) [reply _episodes/01-shell-basics.md]: Your attention span is frighteningly detailed

File: _episodes/01-shell-basics.md
Comment (chrispycheng) [reply _episodes/01-shell-basics.md:257]: I'll keep an eye out!

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: I think you must keep it as a single word since used by the system rendering this whole thing

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: please keep quotes consistent -- the boring `"` instead of `“`

File: _episodes/01-shell-basics.md
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: the same here "SHell" was to describe "SH" within "bash"

File: index.md
Comment (yarikoptic) [inline index.md]: I think that there was some agreement to have here only a paragraph describing what the module is about, for incorporation into moodle or something.  So there should be no opening phrase or "This module will show ...". So may be just keep it here framed as it was for now

**Review by yarikoptic — CHANGES_REQUESTED** [review]
Did a quick pass again. Few minor things... let's finalize these changes and merge and then continue with the other sections, ok?

File: _episodes/02-vcs.md
Comment (yarikoptic) [inline _episodes/02-vcs.md]: well, it is more of "results reproducibility" or that code execution is reproducible and correct

**Review by yarikoptic — COMMENTED** [review]
Just wonderful, thank you!  One little tiny note only spotted 

Happy holidays!

File: _episodes/03-packages.md
Comment (chrispycheng) [inline _episodes/03-packages.md]: This sentence was left incomplete in the original version so please advise on its content! @yarikoptic

File: _episodes/04-legalities.md
Comment (chrispycheng) [inline _episodes/04-legalities.md]: This link is invalid and does not show anonymization tools.

Comment (chrispycheng) [reply _episodes/04-legalities.md]: I also removed the "four aspects" article as it was listed above. Happy to pop it back in if needed

File: _episodes/11-wrap-up.md
Comment (yarikoptic) [inline _episodes/11-wrap-up.md:13]: ha -- cool idea! ;-) did you see it somewhere (so we could extend based on their example) or it just came to your mind?

For further tune up: https://github.com/ReproNim/module-reproducible-basics/issues/22

File: _episodes/05-misc.md
Comment (yarikoptic) [inline _episodes/05-misc.md]: inconsistent `”` sneaked in

File: _episodes/04-legalities.md
Comment (yarikoptic) [reply _episodes/04-legalities.md]: the link is https://open-brain-consent.readthedocs.io/en/latest/anon_tools.html  (has a single `n` in `anon`).  thanks for removing the duplicate and irrelevant links ;-)

File: _episodes/03-packages.md
Comment (yarikoptic) [inline _episodes/03-packages.md]: nah, there is no 10TB file ;) it is 10TB of data in those datasets

File: _episodes/03-packages.md
Comment (yarikoptic) [inline _episodes/03-packages.md]: `pre-existing` is a bad fit here. probably better just to leave "existing" alone

File: _episodes/03-packages.md
Comment (yarikoptic) [reply _episodes/03-packages.md]:
```markdown
supported Debian and Ubuntu releases. If you are using [Singularity](http://singularity.lbl.gov), you could `singularity pull docker://neurodebian[:RELEASE]` to get minimalistic images, or fetch the "Ultimate NeuroDebian Image" by `singularity pull shub://neurodebian/neurodebian`.
```

File: _episodes/03-packages.md
Comment (yarikoptic) [inline _episodes/03-packages.md]: minor -- no need for a space before `,`

File: _episodes/02-vcs.md
Comment (yarikoptic) [inline _episodes/02-vcs.md]: again rogue `“` -- just replace them all. This is a helper I use 
```shell
$> cat bin/git-sedi 
#!/bin/bash

git grep -l "$1" | xargs sed -i -e "s/$1/$2/g"
```
so I would 
1. commit all the changes
2. `git-sedi '“' '"'`
3. verify using `git diff --color-words` that the changes are correct
4. `git commit`

File: _episodes/02-vcs.md
Comment (yarikoptic) [inline _episodes/02-vcs.md]: I am not sure if `initial instance` a good name here... not sure what it means... at least may be `initial invocation` or `once` .

**Review by yarikoptic — CHANGES_REQUESTED** [review]
Thank you very much - it reads much better now.  I left some minor comments around.  Please buzz me whenever it is done -- we should merge this one asap ;-)

File: _episodes/11-wrap-up.md
Comment (chrispycheng) [reply _episodes/11-wrap-up.md:13]: Nah it just came to me - glad you approve! I'm thinking we could definitely get a Google form on there. I'll make a draft and send it your way. I figure it can't hurt to have an extra survey for this specific module even if the whole ReproNim thing has one, right? As for the pull request stuff, I'll brainstorm a bit to see how to get that point across but I'm not sure a simple sentence would be enough, i.e. it wouldn't be effective. What do you think?

Comment (yarikoptic) [reply _episodes/11-wrap-up.md:13]: @Ninja Do we have "Exit surveys" for every topic?  it would be nice to link them here.

Comment (yarikoptic) [reply _episodes/11-wrap-up.md:13]: @chrispycheng Let's just wait for now, we could do it later and meanwhile finalize the fixes/edits

**Comment by yarikoptic** [comment]
Thank you @chrispycheng once again.  I hope that you have also learned a bit while going through it.  I will merge now. Cheers!

**Review by yarikoptic — APPROVED** [review]

**Pull request merged**
